static/src/server.ts

```diff
@@ -271,6 +298,71 @@ function getDisasterBlockTileResults(
   return tilePromises;
 }
 
+// Get the chart html for a tile
+function getTileChart(
+  tileConfig: TileConfig,
+  placeDcid: string,
+  childPlaceType: string,
+  svSpec: StatVarSpec[],
+  eventTypeSpec: Record<string, EventTypeSpec>
+): Promise<string> {
```

Comment:
Instead of returning `strings`, I think it would be more clear to return a`HTMLElement` (or `SVGElement`), and only convert to string right before returning to the user (in the `app.get` handler)

Reply:
done